asv/benchmark.py

```diff
@@ -304,13 +304,15 @@ def get_source_code(items):
     """
     Extract source code of given items, and concatenate and dedent it.
     """
+    from asv.console import log
```

Comment:
not good:
```
$> asv run -j 4 -e -b api.supers.time_ls 3bcc58f^..3bcc58f 
· Creating environments
· Discovering benchmarks
·· Uninstalling from virtualenv-py2.7.
·· Building 3bcc58fa <0.10.3~38> for virtualenv-py2.7.
·· Installing 3bcc58fa <0.10.3~38> into virtualenv-py2.7..
·· Error running /home/yoh/proj/datalad/datalad/.asv/env/1b5862776063636a1bd6a1931ef20cce/bin/python /home/yoh/proj/misc/asv/asv/benchmark.py discover /home/yoh/proj/datalad/datalad/benchmarks /home/yoh/.tmp/tmpTCHmfo/result.json
   STDOUT -------->
   
   STDERR -------->
   Traceback (most recent call last):
     File "/home/yoh/proj/misc/asv/asv/benchmark.py", line 1303, in <module>
       main()
     File "/home/yoh/proj/misc/asv/asv/benchmark.py", line 1296, in main
       commands[mode](args)
     File "/home/yoh/proj/misc/asv/asv/benchmark.py", line 992, in main_discover
       list_benchmarks(benchmark_dir, fp)
     File "/home/yoh/proj/misc/asv/asv/benchmark.py", line 977, in list_benchmarks
       for benchmark in disc_benchmarks(root):
     File "/home/yoh/proj/misc/asv/asv/benchmark.py", line 885, in disc_benchmarks
       class_attr)
     File "/home/yoh/proj/misc/asv/asv/benchmark.py", line 831, in _get_benchmark
       return cls(name, func, sources)
     File "/home/yoh/proj/misc/asv/asv/benchmark.py", line 555, in __init__
       Benchmark.__init__(self, name, func, attr_sources)
     File "/home/yoh/proj/misc/asv/asv/benchmark.py", line 414, in __init__
       self.code = get_source_code([self.func] + self._setups + [self._setup_cache])
     File "/home/yoh/proj/misc/asv/asv/benchmark.py", line 307, in get_source_code
       from asv.console import log
   ImportError: No module named asv.console
```
although `asv` module is available...

Reply:
The `asv` package is not available in the benchmark code.